src/Power.cpp

```diff
@@ -226,7 +227,37 @@ class AnalogBatteryLevel : public HasBatteryLevel
             digitalWrite(ADC_CTRL, LOW);
         }
 #endif
-#else  // ADC2
+#else // ADC2
+#ifdef ADC_CTRL
+#if defined(HELTEC_WIRELESS_PAPER) || defined(HELTEC_WIRELESS_PAPER_V1_0)
+        pinMode(ADC_CTRL, OUTPUT);
+        digitalWrite(ADC_CTRL, LOW); // ACTIVE LOW
+        delay(10);
```

Comment:
Could (should) this delay be shorter? 10ms was originally taken from https://github.com/meshtastic/firmware/blob/05dd680dfa54473f0bbbda1c957c12d956d1802c/src/Power.cpp#L215-L220

Mentioned here: https://github.com/meshtastic/firmware/pull/3163/commits/58acd2809eb2e5148bee2170baef122e6a43605b#r1485661627

Datasheet for the FET: [AO3401](https://aosmd.com/sites/default/files/res/datasheets/AO3401.pdf)

Reply:
I Need to do some measurements with the oscilloscope to evaluate rise/fall time with those resistors more precisely. Unfortunately now I don't have the time to do that, I'll do it in 3/4 weeks.